### PR TITLE
Expectation violation is not an error!

### PR DESCRIPTION
**Pull request by FeepingCreature**

### CODE REVIEW

**Comment by belka-ew** [comment]
Related: https://github.com/funkwerk/dmocks/pull/18#discussion_r382076748

**Comment by FeepingCreature** [comment]
Changed: `Verify` now throws an `Exception`. Expectation error during regular unittest execution still throws an `Error`.

File: source/dmocks/util.d [hunk 1]
```diff
@@ -60,6 +63,17 @@ public class ExpectationViolationError : Error
     }
 }
 
+/**
+ * Thrown when an expectation violation was found during Fixture validation.
```
Comment (linkrope) [inline source/dmocks/util.d:67]: What's "Fixture validation"? Maybe "final verification"?
(`dmocks` does not know what a fixture is.)

Comment (FeepingCreature) [reply source/dmocks/util.d:67]: fixed

**Pull request merged**